packs/data/feats.db/axumas-vigor.json

```
@@ -11,7 +11,7 @@
         },
         "category": "general",
         "description": {
-            "value": "<p>Your vigor grants anyone using Medicine checks to Administer First Aid to you, or use any of the Treat actions on you, a +2 circumstance bonus to their check. During the Long-Term Rest activity, you heal at twice the normal rate.</p>"
+            "value": "<p><strong>Prerequisites</strong> Exposure to the Well of Axuma</p>\n<hr />\n<p>Your vigor grants anyone using Medicine checks to Administer First Aid to you, or use any of the Treat actions on you, a +2 circumstance bonus to their check. During the Long-Term Rest activity, you heal at twice the normal rate.</p>"
```

```suggestion
            "value": "<p>Your vigor grants anyone using Medicine checks to Administer First Aid to you, or use any of the Treat actions on you, a +2 circumstance bonus to their check. During the Long-Term Rest activity, you heal at twice the normal rate.</p>"
```
Already covered by the prereqs field in details.